akkeeper/src/main/scala/akkeeper/master/service/RequestTrackingService.scala

```diff
@@ -28,11 +28,11 @@ private[service] trait RequestTrackingService extends Actor with ActorLogging {
   private var currentServiceReceive: Option[Receive] = None
 
   private def getServiceReceive: Receive = {
-    currentServiceReceive.getOrElse(serviceReceive)
+    currentServiceReceive getOrElse serviceReceive
```

Comment:
What is the motivation behind this update? As far as I can see `getOrElse` is used with dot notation in other places (don't confuse with `orElse` which is a partial function composition).